keyvalue/client/pom.xml

```diff
@@ -35,7 +35,7 @@
     <!-- Internal dependencies -->
     <dependency>
       <groupId>org.alluxio</groupId>
-      <artifactId>alluxio-core-client</artifactId>
+      <artifactId>alluxio-core-client-hdfs</artifactId>
```

Comment:
We provide Hadoop InputFormats and other utilities for working with compute frameworks. In the future I think these can be split into a `hadoop` module for the key value store, but it is not quite the same as the client modules so I didn't attempt to do so.